engine/api/mail/mail.go

```diff
@@ -183,7 +185,12 @@ func SendEmail(ctx context.Context, subject string, mailContent *bytes.Buffer, u
 	headers := make(map[string]string)
 	headers["From"] = smtpFrom
 	headers["To"] = to.String()
-	headers["Subject"] = subject
+	if sdk.StringIsAscii(subject) {
+		headers["Subject"] = subject
+	} else {
+		// https://tools.ietf.org/html/rfc2047
+		headers["Subject"] = "=?UTF-8?Q?" + subject + "?="
```

Comment:
I don't think it is correct.

See `4.2. The "Q" encoding` in https://tools.ietf.org/html/rfc2047, non ascii bytes in `subject` have to be encoded as `=HH`.